subprojects/launcher/src/main/java/org/gradle/tooling/internal/provider/DefaultConnection.java

```diff
@@ -113,7 +113,7 @@ private void assertUsingJava7() {
             UnsupportedJavaRuntimeException.assertUsingVersion("Gradle", JavaVersion.VERSION_1_7);
         } catch (IllegalArgumentException e) {
             // https://github.com/gradle/gradle/issues/3317
-            DeprecationLogger.nagUserWith(e.getMessage(), null, null);
```

Comment:
this is really more a warn message rather than a deprecation message

Reply:
Agreed